mail_activity_to_crm_request/models/mail_activity.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from collections import defaultdict
+from datetime import date, datetime
+from dateutil.relativedelta import relativedelta
+import logging
+import pytz
+import random
+
+from odoo import api, exceptions, fields, models, _
+from odoo.osv import expression
+
+from odoo.tools.misc import clean_context
+from odoo.addons.base.models.ir_model import MODULE_UNINSTALL_FLAG
+
+_logger = logging.getLogger(__name__)
+
+class MailActivity(models.Model):
+    _inherit = 'mail.activity'
+
+    convert_activity_to_sup_req = fields.Boolean(
+        "Convert activity to support request",
+        help="A support request will be created for the activity",
+        default=False,
+    )
+
+    category = fields.Many2one(comodel_name ='crm.claim.category', string='Category')
+    stage = fields.Many2one(comodel_name ='crm.claim.stage', string='Stage')
```

Comment:
The convention for relational field is to suffix them with `_id` or `_ids` depending on the relation type. I suggest that we also add `request` to be more specific in the name. Here it should be this:
```suggestion
    request_category_id = fields.Many2one(comodel_name ='crm.claim.category', string='Category')
    request_stage_id = fields.Many2one(comodel_name ='crm.claim.stage', string='Stage')
```